plugins/remote-config/frontend/public/javascripts/countly.models.js

```diff
@@ -124,23 +131,33 @@
         var parametersResource = countlyVue.vuex.Module("parameters", {
             state: function() {
                 return {
-                    all: []
+                    all: [],
+                    showJsonEditor: false
```

Comment:
this can be a local state within the component i think.

Reply:
Agreed, You can just pass this as a prop to the component

Reply:
as discussed on slack.
this is to be done in subsequent pr's.
just making a note.